howto/container/logs.md

```diff
@@ -1,3 +1,20 @@
+## Runtime logs
+
+At runtime of a container the `amc` commands provides access to both the Anbox and Android system logs. With the following command the logs of both can be viewed:
+
+    amc logs -t <log type>
+
+As `<log type>` you can either specify `anbox` for the Anbox system log or `android` for the Android one. In case that you don't specify the log type by default the Anbox system log will be displayed.
+
+In case that you want to follow the log and get automatically updated for new entries you can add the `-f` argument:
+
+    amc logs -t anbox -f
+
+This will view the logs and update the output whenever new entries are added.
```

Comment:
```suggestion
To follow the log and get automatic updates for new entries, add the `-f` argument:

    amc logs -t anbox -f

This will show the logs and update the output whenever new entries are added.
```